Updates dependencies to fix GHSA-cchq-frgv-rjh5

NSeydoux opened the pull request.

NSeydoux committed:
Updates dependencies to fix GHSA-cchq-frgv-rjh5

jeswr reviewed (approved):
LGTM - lint errors just need fixing

NSeydoux committed:
Uninstall prettier dependency

Prettier comes through the eslint plugin, here the direct dependency was
bringing in an older version and causing issues.

NSeydoux committed:
Lint

NSeydoux committed:
fixup! Lint

NSeydoux committed:
Reinstall prettier

This change will be consistently applied across all repos after a bit
more consideration
